acceptance/help_test.go

```diff
@@ -92,12 +92,13 @@ Usage: om [options] configure-authentication [<args>]
   --version, -v                          bool    prints the om release version (default: false)
 
 Command Arguments:
-  --decryption-passphrase, -dp  string (required)  passphrase used to encrypt the installation
-  --http-proxy-url              string             proxy for outbound HTTP network traffic
-  --https-proxy-url             string             proxy for outbound HTTPS network traffic
-  --no-proxy                    string             comma-separated list of hosts that do not go through the proxy
-  --password, -p, OM_PASSWORD   string (required)  admin password
-  --username, -u, OM_USERNAME   string (required)  admin username
+  --config, -c                  string  path to yml file containing authentication configuration
```

Comment:
The previous help text described what was required and what was optional, I think we need to bring that back in some fashion. Maybe re-add `(required)` next to username and password and change the `config` description to say that it's an alternative to passing the other flags and the config file should contain the flag names as top-level keys.

Reply:
👌